showcase/app/components/shw/divider.gts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { concat } from '@ember/helper';
+
+export interface ShwDividerSignature {
+  Args: {
+    level?: 2;
```

Comment:
with out `Blocks` `glint` would error if you try to splattributes `...attributes`

```
  Blocks: {
    default: [];
  };
```

Reply:
Added, thanks!